charmcraft/templates/init-simple/tox.ini.j2

```diff
@@ -40,7 +40,7 @@ deps =
 commands =
     # uncomment the following line if this charm owns a lib
     # codespell {[vars]lib_path}
-    codespell {toxinidir}/. \
+    codespell {toxinidir} \
```

Comment:
The `tox.ini` files in `init-machine` and `init-kubernetes` probably need this too.

Reply:
Ope, never mind - Jon pointed out that those are symlinks, which my tooling has changed the way it shows so I didn't notice.

Reply:
No, they are symlinks to the "simple" one